packages/nestjs/src/setup.ts

```diff
@@ -53,11 +53,17 @@ class SentryTracingInterceptor implements NestInterceptor {
 
     if (context.getType() === 'http') {
       const req = context.switchToHttp().getRequest();
-      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-      if (req.route) {
-        // eslint-disable-next-line @sentry-internal/sdk/no-optional-chaining,@typescript-eslint/no-unsafe-member-access
+      /* eslint-disable @typescript-eslint/no-unsafe-member-access, @sentry-internal/sdk/no-optional-chaining */
```

Comment:
l: Can we implement this without these eslint overrides? So:

1. avoid optional chaining
2. use type casts & avoid `any` if necessary instead of no-unsafe-member-access

Reply:
I think we can turn off `no-optional-chaining` in the nestjs package, we don't need to worry about the bundle size here.